src/test/app/CFToken_test.cpp

```diff
@@ -0,0 +1,77 @@
+//------------------------------------------------------------------------------
+/*
+  This file is part of rippled: https://github.com/ripple/rippled
+  Copyright (c) 2023 Ripple Labs Inc.
+
+  Permission to use, copy, modify, and/or distribute this software for any
+  purpose  with  or without fee is hereby granted, provided that the above
+  copyright notice and this permission notice appear in all copies.
+
+  THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+  WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+  MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+  ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+  WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+  ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/protocol/Feature.h>
+#include <ripple/protocol/jss.h>
+#include <test/jtx.h>
+
+namespace ripple {
+
+class CFToken_test : public beast::unit_test::suite
+{
+    void
+    testEnabled(FeatureBitset features)
+    {
+        testcase("Enabled");
+
+        using namespace test::jtx;
+        {
+            // If the CFT amendment is not enabled, you should not be able to
+            // create CFTokenIssuances
+            Env env{*this, features - featureCFTokensV1};
+            Account const& master = env.master;
+
+            BEAST_EXPECT(env.ownerCount(master) == 0);
+
+            env(cft::create(master, "0158415500000000C1F76FF6ECB0BAC600000000"),
+                ter(temDISABLED));
+            env.close();
+
+            BEAST_EXPECT(env.ownerCount(master) == 0);
+        }
+        {
+            // If the CFT amendment IS enabled, you should be able to create
+            // CFTokenIssuances
+            Env env{*this, features | featureCFTokensV1};
+            Account const& master = env.master;
+
+            BEAST_EXPECT(env.ownerCount(master) == 0);
+
+            env(cft::create(
+                master, "0158415500000000C1F76FF6ECB0BAC600000000"));
```

Comment:
we're _NOT_ going to do this, as we discussed on the call with fuelling. i nealry completed an implementation like you're describing, but i don't think it's worth it

Reply:
I suggested in one of the comments to combine `sfIssuer` and `sfAssetCode` since there is `SF_ISSUE` defined in develop. `SF_CURRENCY` could work too. Why you decided not to do this? Why `sfAssetCode` is different? Isn't it the same as what is used in `STAmount`?